Cant resolve webpack Aliases with mocha

Now the big problem is that i cant resolve webpack aliases with mocha or mocha webpack. 
I have aliases of this sort in my webpack 

```
alias: {
       LoginActions : path.resolve(__dirname, 'app/actions/login-actions.jsx/'),
}
```
in my components which are not tests i import it like this

`import LoginActions from 'LoginActions'`

this is fine .. but the problem comes when i run tests on these modules .  My test case is here.

```
import {assert} from "chai";
 import {emailChangeAction,loginAction} from "LoginActions";

 describe("LoginActions",()=>{
    describe('changeEmailAction',()=>{
      it('Should generate changemail Action',()=>{
          const action ={
            type :'EMAIL_CHANGED',
            email:'bbosa.henry1@gmail.com'
          }
          const res = emailChangeAction(action.email);
          assert.equal(res,action);


      })

    });

 });
```

in my console i get this error 
`Module not found: 'LoginActions' in 'C:\projects\aurity\reactweb\test\actions'`

in my package.json i have this script:

```
scripts :{
     "test": "mocha-webpack  --recursive \"test/**/*.@(js|jsx)\"",
}
```

Now how do i test with with these aliases. Because i use them alot in my project

--- a/src/cli/parseArgv.js
+++ b/src/cli/parseArgv.js
@@ -171,7 +171,7 @@ const options = {
     describe: 'path to webpack-config file',
     group: BASIC_GROUP,
     requiresArg: true,
-    default: 'webpack-config.js',
+    default: 'webpack.config.js',
   },
   'webpack-env': {
     describe: 'environment passed to the webpack-config, when it is a function',

--- a/test/unit/cli/parseArgv.test.js
+++ b/test/unit/cli/parseArgv.test.js
@@ -805,7 +805,7 @@ describe('parseArgv', function () {
         const parsedArgv = this.parseArgv(argv);
 
         // then
-        assert.propertyVal(parsedArgv, 'webpackConfig', 'webpack-config.js');
+        assert.propertyVal(parsedArgv, 'webpackConfig', 'webpack.config.js');
       });
 
 

